추가기능요구사항.md

```
@@ -0,0 +1,256 @@
+- [x] userId를 조건으로 조회합니다.
+
+### update
+
+- [x] DTO를 활용해 파라미터를 그룹화합니다.
+  수정 대상 객체의 id 파라미터, 수정할 값 파라미터
+
+### delete
+
+- [x] id로 삭제합니다.
+
+--- 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)